src/locker/locks/tests/passphrase.test.js

```diff
@@ -0,0 +1,282 @@
+import createPassphraseLock from '../passphrase';
+import { secret, storedPassphrase, crypto, TextEncoder, enableTest, unlockTest } from './mocks';
+
+const mockSecret = {
+    generate: jest.fn(() => secret),
+    get: jest.fn(() => secret),
+    set: jest.fn(),
+};
+
+global.crypto = crypto;
```

Comment:
Question: is `crypto` and `TextEncoder` unavailable in Jest?

I'm asking that because we could use `jest.fn` per test instead of relying in a separate "mocks" fns.

Reply:
Yes, both crypto and TextEncoder are unavailable in jest.
I want to look at this with more detail. Maybe discuss with you in further detail afterwards. Is that ok?

Reply:
Alright!